src/adhocracy_mercator/adhocracy_mercator/scripts/export_users.py

```diff
@@ -7,16 +7,61 @@
 import sys
 import textwrap
 import time
-
 from pyramid.paster import bootstrap
 from substanced.util import find_catalog
 
+from adhocracy_core.catalog.adhocracy import index_rates
+from adhocracy_core.resources.principal import IUser
+from adhocracy_core.sheets.metadata import IMetadata
 from adhocracy_core.sheets.principal import IUserBasic
 from adhocracy_core.sheets.principal import IUserExtended
-from adhocracy_core.sheets.metadata import IMetadata
+from adhocracy_core.sheets.rate import IRateable
+from adhocracy_core.sheets.rate import IRate
 from adhocracy_core.utils import get_sheet_field
+from adhocracy_mercator.resources.mercator import IMercatorProposalVersion
+from adhocracy_mercator.sheets.mercator import ITitle
+from adhocracy_core.sheets.tags import filter_by_tag
+
+
+def _get_most_rated_proposals(root, min_rate=100):
```

Comment:
The code would be simpler if you use the pool sheet to search the catalog. 
You can add all filter prameters describes in rest_api.rst  to the pool sheet get method.